meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/util/ZkMetaClientUtil.java

```diff
@@ -235,6 +237,8 @@ public static MetaClientException translateZkExceptionToMetaclientException(ZkEx
       return new MetaClientTimeoutException(e);
     } else if (e instanceof ZkInterruptedException) {
       return new MetaClientInterruptException(e);
+    } else if (e instanceof ZkNodeExistsException) {
```

Comment:
Where is this exception used?

Reply:
For puppy stress testing in future PR's (note in PR description)